app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -528,7 +530,8 @@ class MapActivity : ComponentActivity() {
     }
 
     /**
-     * The HUD for the player stats
+     * The HUD for the player stats, it displays basic information such as their balance,
+     * and a button shows complete information on click
```

Comment:
you can precise the description of the param in the documentation of the function with `@param`!

Reply:
Thanks for the suggestion!

Reply:
fyi, "préciser" en français -> "to specify" in English, precise is only an adjective ^^